test/WebJobs.Script.Tests.Integration/Management/InstanceManagerTests.cs

```diff
@@ -136,12 +136,15 @@ public async Task ValidateContext_InvalidZipUrl_ReturnsError()
             };
 
             string error = await _instanceManager.ValidateContext(assignmentContext);
-            Assert.Equal("Invalid zip url specified (StatusCode: NotFound)", error);
+            Assert.Equal("Invalid zip url specified", error);
```

Comment:
Wasn't the previous error message which included status code more informative? Why reduce?

Reply:
Fixed